packages/select/stories/select.stories.tsx

```diff
@@ -33,183 +29,210 @@ export default {
     required: true,
     clearButtonText: 'clear',
     clearButtonProps: {},
+    label: 'Field Label',
   },
-} as Meta;
+};
 
-export const Default: Story = ({
-  autofill,
-  creatable,
-  disabled,
-  helpMessage,
-  isMulti,
-  isClearable,
-  max,
-  min,
-  raw,
-  required,
-  clearButtonText,
-  clearButtonProps,
-}) => (
-  <FormikResults
-    onSubmit={() => {
-      console.log('submitted');
-    }}
-    initialValues={{
-      select: undefined,
-      autoFill1: '',
-      autoFill2: '',
-    }}
-    validationSchema={
-      isMulti
-        ? multiValueSchema('select', required, min, max, !autofill && !raw)
-        : singleValueSchema('select', required, !autofill && !raw)
-    }
-  >
-    <Row>
-      <Col>
-        {autofill ? (
-          <>
+type SelectStoryProps = {
```

Comment:
is there a reason we're creating a new type instead of inferring the type? `StoryObj<typeof Select>`